feature-libs/order/core/store/effects/order-by-id.effect.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpErrorResponse } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { normalizeHttpError } from '@spartacus/core';
+import { Order } from '@spartacus/order/root';
+import { Observable, of } from 'rxjs';
+import { catchError, concatMap, map } from 'rxjs/operators';
+import { OrderHistoryConnector } from '../../connectors/order-history.connector';
+import { OrderActions } from '../actions/index';
+
+@Injectable()
+export class OrderByIDEffect {
```

Comment:
[MINOR]
For consistency, let's use camelCase for `Id` (instead of `ID`).

This remark applies also to other places in this PR

Reply:
done